app/views/admin/timelogs/index.html.erb

```diff
@@ -0,0 +1,27 @@
+<div class="well">
+  <% if params[:timelogs] == 'false' %>
+    <h4>This user have no timelogs yet</h4>
+    <%= link_to 'Create timelog', new_admin_user_timelog_path(params[:user_id]), class: 'btn btn-default' %><br>
+  <% else %>
+    <h3>Timelogs</h3>
+    <strong>Filter by:</strong><p><%= link_to "Start time", admin_user_timelogs_path(:filter=>"start_time") %>
```

Comment:
please, try to avoid old hash styles. It is a good practice to write clean and formatted code from the very beginning.
`The less code you have the less bugs you have`